packages/x-data-grid/src/models/gridStateCommunity.ts

```diff
@@ -62,6 +62,7 @@ export interface GridStateCommunity {
   virtualization: GridVirtualizationState;
   columnResize: GridColumnResizeState;
   rowSpanning: GridRowSpanningState;
+  listView: boolean;
```

Comment:
If it's just mirroring the prop, we can put it in "state props" instead:

https://github.com/mui/mui-x/blob/7ab76f0428b2f49d72ee367201aeff7c6ce1b1d6/packages/x-data-grid/src/hooks/core/useGridProps.ts#L26-L28